vtm/src/org/oscim/core/MapElement.java

```diff
@@ -75,11 +75,18 @@ public String toString() {
     public MapElement clone() {
         MapElement copy = new MapElement();
```

Comment:
If clone the filled array parts, then better create the `MapElement` with the correct arrays size via `MapElement(int points, int indices)` constructor, to avoid unneeded space overhead.

Reply:
yes, good idea

Reply:
I added a constructor, so no new arrays were created.